redis/cache/redisx/redisxtest/redisxtest.go

```diff
@@ -0,0 +1,39 @@
+package redisxtest
+
+import (
+	"context"
+	"testing"
+
+	"github.com/joomcode/redispipe/redis"
+	"github.com/joomcode/redispipe/redisconn"
+
+	"github.com/reddit/baseplate.go/redis/cache/redisx"
+)
+
+// NewMockRedisClient sets up a redis client for use in testing
+func NewMockRedisClient(
+	ctx context.Context,
+	tb testing.TB,
+	address string,
+	opts redisconn.Opts,
+) (*redisx.BaseSync, error) {
+	tb.Helper()
+
+	// Create connection to redis
+	conn, err := redisconn.Connect(ctx, address, opts)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
oh and this function no longer need to return an error. this can be just:
```suggestion
	if err != nil {
		tb.Fatalf("Failed to create redis client: %v", err)
	}
```